packages/blockchain-wallet-v4-frontend/src/scenes/Settings/Addresses/index.js

```diff
@@ -1,11 +1,12 @@
 import { Redirect, Route, Switch, withRouter } from 'react-router-dom'
+import React from 'react'
+import styled from 'styled-components'
+
 import Bch from './Bch'
 import Btc from './Btc'
 import BtcManage from './Btc/ManageAddresses'
 import Eth from './Eth'
 import Menu from './Menu'
-import React from 'react'
-import styled from 'styled-components'
```

Comment:
(minor) - we can easily migrate this file to TS

Reply:
yeah i thought about it. but i think it can wait until the pages get the true redesign it needs